tests/plenary/simple_busted_spec.lua

```diff
@@ -17,6 +19,34 @@ describe('busted specs', function()
     pcall(tester_function)
   end)
 
+  describe('before each clear, clears the lines', function()
+    before_each(clear)
+    it('inserts some text', function()
+      eq({''}, vim.api.nvim_buf_get_lines(0, 0, -1, false))
+      vim.api.nvim_buf_set_lines(0, 0, -1, false, { 'foo bar baz' })
+      eq({'foo bar baz'}, vim.api.nvim_buf_get_lines(0, 0, -1, false))
+    end)
+
+    it('no more content in the buffer', function()
+      eq({''}, vim.api.nvim_buf_get_lines(0, 0, -1, false))
+    end)
+    -- Should we also clear here?
+    -- Before if the last tests put something in the buffer,
+    -- the next running test will have it in the buffer
```

Comment:
I think we should not clear here. The user should just call clear if he leaves the `describe`.